src/coreclr/jit/lsraxarch.cpp

```diff
@@ -147,6 +147,7 @@ int LinearScan::BuildNode(GenTree* tree)
         case GT_CNS_INT:
         case GT_CNS_LNG:
         case GT_CNS_DBL:
+        case GT_CNS_VEC:
```

Comment:
Noting ARM64 already had a path here since it needed special handling for `Zero` and `AllBitsSet`, it just wasn't getting used in LSRA